src/e2e-tests/src/intTest/resources/behavior/ui/csAddMember.feature

```diff
@@ -8,6 +8,9 @@ Feature: Central Server Add Member
     And User xrd logs in to CentralServer with password secret
 
   Scenario: Member is created and listed
-    Given Members tab is selected
-    When A new member is added
+    Given CentralServer Settings tab is selected
+    And System settings tab is selected
+    And A new memberclass XXX is added
+    And Members tab is selected
+    When A new member with XXX memberclass is added
     Then A new member is listed
```

Comment:
"is listed" checks for count, this can get flaky if someone else adds new member during test execution

Reply:
Would adding with e2e-specific code, class & name values and later checking if that combination exists in the table be sufficient?

Reply:
I would think so, we cannot add duplicates right?

Reply:
class-code combination has to be unique yes